File: server/src/weblab/data/dto/ExternalEntity.py
#!/usr/bin/env python
#-*-*- encoding: utf-8 -*-*-
#
# Copyright (C) 2005-2009 University of Deusto
# All rights reserved.
#
# This software is licensed as described in the file COPYING, which
# you should have received as part of this distribution.
#
# This software consists of contributions made by many individuals, 
# listed below:
#
# Author: Jaime Irurzun <jaime.irurzun@gmail.com>
#

class ExternalEntity(object):
    
    def __init__(self, name, country, description, email, id=None):
        super(ExternalEntity,self).__init__()
        self.id = id
        self.name = name
        self.country = country
        self.description = description
        self.email = email

    def __repr__(self):
        return "ExternalEntity(id = %i, name = '%s', country = '%s', description = '%s', email = '%s')" % (
                self.name,
                self.country,
                self.description,
                self.email
            )

File: client/src/es/deusto/weblab/client/dto/users/ExternalEntity.java
/*
* Copyright (C) 2005 onwards University of Deusto
* All rights reserved.
*
* This software is licensed as described in the file COPYING, which
* you should have received as part of this distribution.
*
* This software consists of contributions made by many individuals, 
* listed below:
*
* Author: Jaime Irurzun <jaime.irurzun@gmail.com>
*
*/

package es.deusto.weblab.client.dto.users;

public class ExternalEntity extends Agent {
	
	private int id;
	private String name;
	private String country;
	private String description;
	private String email;
	
	public ExternalEntity() {}
	
	public ExternalEntity(int id, String name, String country, String description, String email) {
		this.id = id;
		this.name = name;
		this.country = country;
		this.description = description;
		this.email = email;
	}

	public int getId() {
		return this.id;
	}
	public void setId(int id) {
		this.id = id;
	}
	public String getName() {
		return this.name;
	}
	public void setName(String name) {
		this.name = name;
	}
	public String getCountry() {
		return this.country;
	}
	public void setCountry(String country) {
		this.country = country;
	}
	public String getDescription() {
		return this.description;
	}
	public void setDescription(String description) {
		this.description = description;
	}
	public String getEmail() {
		return this.email;
	}
	public void setEmail(String email) {
		this.email = email;
	}
}
